Ingest.Dockerfile

```diff
@@ -10,16 +10,14 @@ RUN apt-get update -y && \
     apt-get install -y build-essential make git
 COPY digital_asset_types /rust/digital_asset_types
 RUN mkdir -p /rust/nft_ingester
-WORKDIR /
-RUN git clone https://github.com/metaplex-foundation/blockbuster
 WORKDIR /rust/nft_ingester
 COPY --from=planner /rust/nft_ingester/recipe.json recipe.json
 # Build dependencies - this is the caching Docker layer!
 COPY nft_ingester/Cargo.toml .
 RUN cargo chef cook --release --recipe-path recipe.json
 COPY nft_ingester .
 # Build application
-RUN cargo build --release
+RUN RUSTFLAGS=-Awarnings cargo build --release
```

Comment:
i think a separate cleanup pr will be good for this. any kind of green underline warning causes the build to fail in docker. i will make a linear task for cleanup work so that this can be removed.